packages/auth/src/core/auth/initialize.ts

```diff
@@ -85,3 +90,24 @@ export function _initializeAuthInstance(
   // eslint-disable-next-line @typescript-eslint/no-floating-promises
   auth._initializeWithPersistence(hierarchy, deps?.popupRedirectResolver);
 }
+
+export async function _initializeRecaptchaConfig(auth: Auth): Promise<void> {
```

Comment:
Can we put this as a static method on RecaptchaEnterpriseVerifier instead? https://github.com/firebase/firebase-js-sdk/blob/960f109970713f7a80c2649b02b1154b7c743bed/packages/auth/src/platform_browser/recaptcha/recaptcha_enterprise_verifier.ts#L38

This core/auth/initialize.ts has logic related to initializing auth instance itself, so adding other feature initialization seems heavyweight.

Reply:
hmm, that's a good point. I was thinking putting all initialize method to the initialize.ts file, which should still allow tree shaking, but seems recaptcha_enterprise_verifier.ts is a better place.